packages/cerebral/src/Controller.js

```diff
@@ -33,6 +31,10 @@ class Controller extends EventEmitter {
     }, this)
     this.router = router ? router(this) : null
 
+    if (options.strictRender) {
+      console.warn('DEPRECATION - No need to use strictRender option anymore, it is the only render mode now')
```

Comment:
Users of non strict mode would be unwarned about change.
Would be automatic "strict" warnings would be enough?
Maybe just warn everyone until release?